src/PeriodEnd/PaymentsDue/DeployPaymentsDue/sql/dml/14 PeriodEnd.PaymentsDue.PreRun.Staging.ApprenticeshipEarnings3.sql

```diff
@@ -96,4 +99,21 @@ INSERT INTO Staging.ApprenticeshipEarnings3
 				(COALESCE(pepm.TransactionTypesFlag, 1) = 1  And ndtt.TransactionType = 15 AND ade.LearningSupportPayment <> 0 )
 			)
 GO
-GO
\ No newline at end of file
+
+IF EXISTS (
+		SELECT 1
+		FROM [sys].[indexes] i
+		JOIN sys.objects t ON i.object_id = t.object_id
+		WHERE t.name = 'ApprenticeshipEarnings3'
+		AND i.[name] = 'IX_ApprenticeshipEarnings3_UKPRN'
+		)
+BEGIN
+	DROP INDEX [IX_ApprenticeshipEarnings3_UKPRN]
+		ON [Staging].[ApprenticeshipEarnings3]
+END
+
+CREATE CLUSTERED INDEX [IX_ApprenticeshipEarnings3_UKPRN] ON [Staging].[ApprenticeshipEarnings3]
+(
+	[Ukprn] ASC
+)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, SORT_IN_TEMPDB = OFF, DROP_EXISTING = OFF, ONLINE = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
+GO
```

Comment:
there is a step in the payments process that indexes these three tables and rather than vw_apprenticeshipearning looking at apprenticeshipearning1, 2, 3, it looks at a new table that contains all the data from the three tables.

Therefore, remove index.